### ISSUE TITLE
All global message notifications should be vocalized as soon as they happen

### ISSUE THREAD

**Issue by Larisa-Staroverova** [SAP/spartacus#13679]
Should be adjusted `product-configurator-conflict-and-error-messages.component.html` accordingly

**Comment by Larisa-Staroverova** [comment]
See also #13677
